homework-week3.js

```diff
@@ -0,0 +1,196 @@
+// 1. Strings!
+
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);   //hello,this,is,a,difficult,to,read,sentence
+console.log(myString.length);   //42
+
+//Find a way to remove the commas from the string and replace them with spaces.
+for (let i=0; i<myString.length; i++){
```

Comment:
While the output is correct, there is no need to loop over the length of the string. If you look in the console log the output is 42 times the same string. Removing the loop will result in the same output.